src/language/en-US.json

```diff
@@ -169,7 +169,9 @@
     "duration_can_not_more_than_10_years": "Duration cannot be greater than 10 years",
     "duration_permanent": "Unlimited duration",
     "end_finalization_epoch": "End Finalization Epoch",
+    "encrypt_message": "Encrypt message",
```

Comment:
Probably it's no big deal but worth to mention here; I prefer to have all the keys sorted alphabetically so that we can find a key easily and avoid any duplicates. I use this plugin on VSCode => https://marketplace.visualstudio.com/items?itemName=richie5um2.vscode-sort-json . And also for the translations using this one, highly recommended => https://marketplace.visualstudio.com/items?itemName=antfu.i18n-ally